bin/test-example

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
👍 

One thing that I was not sure on was whether `/bin/sh` is OK to leave intact, but my quick research tells me that it is (please let me know if I was wrong).

Actually I will say, I would rather have scripts use `/bin/sh` rather than `bash`, so it would be best if it would not use bash-specific features. I believe the only one is the default-var. I'll probably add slightly more verbosity.